linked_list/c/1.linked list/160101029_1_1.c

```diff
@@ -0,0 +1,221 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+//structure representing linked list
+struct node
+{
+	char d;
+	struct node *next;
+};
+
+
+struct node *head1;			//head pointer for list1
+struct node *head2;			//head pointer for list 2
+
+
+
+int count1,count2;		//variables to count no. of comparisions
+
+
+
+//move to front function
+void move_to_front(char alpha)
+{
+	struct node *temp=(struct node *)malloc(sizeof(struct node));		//creating a temporary node
+	//storing alpha in temporary node
+	temp->d=alpha;	
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.cpp`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpdqz8lpse/linked_list/c/1.linked list/160101029_1_1.c
+++ b/tmp/tmpdqz8lpse/linked_list/c/1.linked list/160101029_1_1.c
@@ -23,7 +23,7 @@
 {
 	struct node *temp=(struct node *)malloc(sizeof(struct node));		//creating a temporary node
 	//storing alpha in temporary node
-	temp->d=alpha;	
+	temp->d=alpha;
 	temp->next=NULL;
 
 
```